backup-environment/action.yml

```diff
@@ -5,7 +5,15 @@ description: "Power Platform Backup Environment Action"
 inputs:
   environment-url:
     description: 'URL of Power Platform environment to backup; e.g. "https://test-env.crm.dynamics.com"'
-    required: true
+    required: false
+
+  environment-id:
```

Comment:
no need to introduce the -id action parameter, as it would eventually also need to be obsoleted...
the right pattern is: make env-url optional with just the environment parameter which accepts both url and id.